gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/ChannelPool.java

```diff
@@ -152,6 +152,13 @@ Channel getChannel(int affinity) {
   /** {@inheritDoc} */
   @Override
   public ManagedChannel shutdown() {
+    if (LOG.isLoggable(Level.FINE)) {
+      // Synthetic exception to generate a stacktrace in the logs
+      RuntimeException e =
+          new RuntimeException("Initiating graceful shutdown due to explicit request");
```

Comment:
This seems pretty misleading. It will literally create an error report in Error Reporting.

Reply:
removed stacktrace